src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -1514,6 +1517,108 @@ def handle_assign_role_submission_event(
     )
 
 
+def handle_create_task_command(
+    ack: Ack,
+    body,
+    dict,
+    client: WebClient,
+    context: BoltContext,
+    db_session: Session,
+) -> None:
+    """Displays a modal for task creation."""
+    ack()
+    if context["subject"].type == CaseSubjects.case:
+        raise CommandError("Command is not currently available for cases.")
```

Comment:
suggestion: we should return this message as an 'error modal' as this will cause an 'unexpected error' state.

I believe there's a pattern for this already somewhere in the `interactive.py` files.

Reply:
Done.